### PR TITLE
Changed so that the CSS kicks in when the width drops below 1024 px

### LINKED ISSUE #3437: Refactor CSS for Home Page Footer on Smaller Screens for Redesigned Homepage
### Overview
Currently, when you compress the HfLA home page on a computer screen, the CSS for mobile doesn't activate. We need to update the CSS so that the mobile CSS is activated for the compressed Home Page to ensure the intended formatting.

### Action Items
- [x] Feature Branch: You will be working off a feature branch called feature-homepage-launch. Please follow ["Step 1: Create your branch" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-1-create-your-branch) wiki article.
- [x] Review [_footer.scss](https://github.com/hackforla/website/blob/feature-homepage-launch/_sass/components/_footer.scss) and [_footer.html](https://github.com/hackforla/website/blob/feature-homepage-launch/_includes/footer.html) to understand how CSS is applied to the footer
- [x] Change the CSS so that the footer displays like the mobile version on compressed screens
- [x] Confirm everything is working as intended
- [x] To update your topic branch, please follow ["Step 2: Update your branch with upstream changes" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-2-update-your-branch-with-upstream-changes) wiki article.
- [x] The pull request created from this issue should be merged into feature-homepage-launch. Please follow ["Step 3: Create a pull request" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-3-create-a-pull-request) wiki article

### Resources/Instructions
- Current redesigned Home page
  <details>
    <summary>Click here to see the current redesigned Home page as it is being compressed from 1024 to smaller sizes in developer tools:</summary>

    redesigned-homepage-footer-compression problem

  </details>

- Desired Home page for smaller screens:
intended home page

Files:
- [_footer.scss](https://github.com/hackforla/website/blob/feature-homepage-launch/_sass/components/_footer.scss) 
- [_footer.html](https://github.com/hackforla/website/blob/feature-homepage-launch/_includes/footer.html) 
- [How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch)
- [Homepage Redesign in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17951%3A122724)

### PR DESCRIPTION
**Pull request by h3nry-m**
Fixes #3437 

### What changes did you make and why did you make them ?

  - Changed the media screen to encompass widths that were less than 1024px so that the CSS now works as soon as the screen size reduces as opposed to a gradual reduction in screen size.

### CODE REVIEW

**Comment by roslynwythe** [comment]
Availability: 11/29 10 pm - midnight, 11/30 10 am - noon
ETA: 11/30

File: _sass/components/_footer.scss [hunk 1]
```diff
@@ -18,64 +15,63 @@
...
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    margin: 0;
+  }
+  .main-footer.footer-links {
+    justify-content: space-between;
+    margin: 0;
+  }
+  .main-footer.footer-columns {
```
Comment (roslynwythe) [inline _sass/components/_footer.scss:61]: Did you intend this selector to be .main-footer .footer-column ? Note that the space in-between the class names changes the meaning of the selector.

**Comment by roslynwythe** [comment]
[h3nry-m](https://github.com/h3nry-m) Great work re-organizing the media queries into one section of _footer.scss, and applying the new rule:

`screen and (max-width: 1024px)`

It is a best practice to use sass variables to represent media query breakpoints,  They can be found in _sass\variables\_layout.scss, but I do not see any for 1024px so I'm curious where the breakpoint of 1024px came from.   

Testing your CSS in chrome dev tools, I cannot confirm that for screens < 1024px, the styles contained in the media query are actually applied.  From the screenshot it appears that there should be 3 columns of links at wide screens but I don't see those three columns.  I believe that the reason I see no change at 1024px is that within the media query, the CSS selectors do not apply to the appropriate elements.  Please see my specific code suggestion/comment at line 61.

**Comment by esantiano** [comment]
ETA: 12/08/2022
Availability: 7pm - 9pm PST Thursday

**Comment by h3nry-m** [comment]
Hi @roslynwythe thanks for the review! Sorry it's taken me so long to get back to you. For the breakpoint, I got 1024px from the description - I made the assumption that they wanted the sizes to change starting at 1024px. You are right, I missed the space between the CSS selectors and did not identify them! I plan on working on this tomorrow and if I can't finish then I'll stop by office hours on Wednesday. 

PS. I see that in variables_layout.scss it says that screen-mobile is 480px. Is that the correct breakpoint?

ETA: 12/7/22 Wednesday
Availability: 6pm-9pm

**Review by arpitapandya — CHANGES_REQUESTED** [review]
@h3nry-m Looks like you've made changes in feature-homepage-launch directly. Please create a branch [follow](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-1-create-your-branch) 


createatopicbranch

**Comment by roslynwythe** [comment]
@Zak234 @JessicaLucindaCheng please clarify the desired responsive behavior.  The issue resources shows a video of the website compressed, starting from 1024px.  Please advise, what is the breakpoint at which the 3 column design is supposed to change to 2 columns?

**Comment by JessicaLucindaCheng** [comment]
@roslynwythe 

> @Zak234 @JessicaLucindaCheng please clarify the desired responsive behavior. The issue resources shows a video of the website compressed, starting from 1024px. Please advise, what is the breakpoint at which the 3 column design is supposed to change to 2 columns?

As far as I know, I don't think designers have specified the breakpoint between desktop and mobile. Thus, I would probably use the breakpoint variable `$bp-below-desktop`, which is defined in [_sass/variables/_layout.scss](https://github.com/hackforla/website/blob/gh-pages/_sass/variables/_layout.scss). If that looks weird, you can try one of the other breakpoint variables in the _layout.scss file

**Comment by roslynwythe** [comment]
> Hi @roslynwythe thanks for the review! Sorry it's taken me so long to get back to you. For the breakpoint, I got 1024px from the description - I made the assumption that they wanted the sizes to change starting at 1024px. You are right, I missed the space between the CSS selectors and did not identify them! I plan on working on this tomorrow and if I can't finish then I'll stop by office hours on Wednesday.
> 
> PS. I see that in variables_layout.scss it says that screen-mobile is 480px. Is that the correct breakpoint?
> 
> ETA: 12/7/22 Wednesday Availability: 6pm-9pm

Hi @h3nry-m I believe that you will probably require only one media query/breakpoint, and in Jessica's comment, she suggests that breakpoint should be $bp-below-desktop, which is defined in [_sass/variables/_layout.scss](https://github.com/hackforla/website/blob/gh-pages/_sass/variables/_layout.scss).  Please see her comment.

**Comment by roslynwythe** [comment]
@arpitapandya it looks like there is another issue# [2393 ](https://github.com/hackforla/website/issues/2393#issue-1034084612) which is also updating the footer CSS.  It looks like this issue is focusing on mobile and the other on desktop.  Is that a typical strategy?  It seems like it may result in confusion during merge - please advise.

**Comment by jyaymie** [comment]
I'm currently working on issue [#2393 ](https://github.com/hackforla/website/issues/2393#issue-1034084612) that @roslynwythe was wise in pointing out. I've been using `$bp-mobile-up` to update the desktop footer, but I can rework my code depending on what breakpoint variable we decide to move forward with.

I also noticed that the bottom links on the Figma design for mobile `Credits | Site Map | Join Us` differ from those for desktop `Credits | Privacy Policy | Site Map`. Should the links be the same or dynamically changed?

**Comment by h3nry-m** [comment]
Hi all, unfortunately I have some obligations now that I can no longer ignore and am unable to continue volunteering with Hack for LA. Should I close this pull request?

**Comment by arpitapandya** [comment]
@h3nry-m If you think you can not contribute, it's totally fine. You may close Pull Request. Once PR is closed I will un-assign you from the issue and I will move it to `prioritized backlog` so that any other dev can work on it and open a new PR. Thank you for your contributions to the project.

**Pull request closed without merging**